source/includes/harvest/_scheduled_interviews.md

```diff
@@ -303,3 +326,235 @@ Retrieve an interview by its ID.
+      "scorecard_id": null
+    },
+    {
+      "id": 46444,
+      "employee_id": "AAA1",
+      "name": "Lucius Barbarossa",
+      "email": "lucius.barbarossa.46444@example.com",
+      "response_status": "declined",
+      "scorecard_id": null
+    }
+  ]
+}
+```
+
+Create a new Scheduled Interview. 
+
+[See noteworthy response attributes.] (#the-scheduled-interview-object)
+
+### HTTP Request
+
+`POST https://harvest.greenhouse.io/v1/scheduled_interviews`
+
+### Headers
+
+Header | Description
+--------- | -----------
+On-Behalf-Of | ID of the user issuing this request. Will become the interview organizer. The user must have permission to manage the associcated application.
+
+### JSON Body Parameters
+
+Parameter | Required | Type | Description
+--------- | ----------- | ----------- | -----------
+application_id | Yes | integer | The id of the application for which we will scheduled an interview.
+interview_id | Yes | integer | The id of the interview we'd like to schedule. This id can be found in the [job stage object](#the-job-stage-object). It refers to an interview type (e.g. "Executive Interview" or "On-site Interview") that is associated with a hiring plan's interview step.  This differs from a "Scheduled Interview" which refers to an instantiated interview that is associated with an application, interviewers, start time, etc.
+interviewers[] | Yes | interviewer | Array of interviewers. Each must specify a user by user_id, email, or employee_id. Each must include a response status (one of needs_action, declined, tentative, or accepted). Each user must be a valid interviewer for the hiring plan.
+start | Yes | string | A datetime specifying when the interview starts. Must be provided in [ISO-8601](#general-considerations) format (e.g. 2018-11-05T13:12:14Z).
+end | Yes | string | A datetime specifying when the interview ends. Must be provided in [ISO-8601](#general-considerations) format (e.g. 2018-11-05T13:12:14Z).
+external_event_id | Yes | string | A unique identifer for this interview.
+location| No | string | A textual description of the location of the interview.
+
+<aside class="notice">
+    There may be a delay between when Greenhouse receives the POST: Create Scheduled Interview request and when Greenhouse creates the full Scheduled Interview record, which will result in a truncated API response. The truncated response body will contain the id of the newly Scheduled Interview. You can retrieve the full Scheduled Interview record by requesting the Scheduled Interview ID with the GET: Scheduled Interview endpoint. If you receive a 404 error from the GET: Scheduled Interview endpoint, this indicates that the full Scheduled Interview record is still not available. Until the Scheduled Interview record has been made fully-available in the API, please continue to request the record until the API returns a successful response. Our recommendation is to perform this check every 30 seconds until the data becomes available.
+</aside>
```

Comment:
What causes this to happen?

Reply:
so we have to create the model through GHR, and then GHR tells harvest "okay this record is created here's the ID." When harvest gets this "OK" from GHR, it uses the provided ID to look it up in the follower DB.  However, the created record may not have been propagated to the follower DB yet, in which case it won't be able to return the newly created record.  In these scenarios we'll return a 202 to indicate that the process was successful but the result is not yet available.

Reply:
Ah, ok